app/utils/no_cache.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-module.exports = function (res) {
+module.exports = (res) => {
```

Comment:
Small thing but if you only have one variable you don't need the brackets

So can be `module.exports = res => {...}`

Reply:
Done